plugins/ohif/nvidia.pem

```diff
@@ -0,0 +1,178 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
possibly not a good idea to save the cert etc.. see if ignore ssl is enough in this case..

Reply:
Ok, i think the cert is not critical anymore, I will try remove it and just turn off SSL verify.